miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/EditSampleController.java

```diff
@@ -694,14 +694,13 @@ public ModelAndView setupForm(@PathVariable Long sampleId, @PathVariable Long pr
 
       model.put("formObj", sample);
       model.put("sample", sample);
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "sample" 4 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)